packages/runtime/src/decorators/response.ts

```diff
@@ -23,6 +24,18 @@ export function Response<ExampleType, HeaderType extends IsValidHeader<HeaderTyp
  * The type of the responder function should be annotated `TsoaResponse<Status, Data, Headers>` in order to support OpenAPI documentation.
  */
 export function Res(): Function {
+  // todo: allow setting media type
```

Comment:
I think it'd be better to check the Header type (3rd arg to `TsoaResponse`) over adding an arg to `@Res`.